src/components/AddressSearch.js

```diff
@@ -86,10 +87,17 @@ const AddressSearch = (props) => {
         <GooglePlacesAutocomplete
             ref={googlePlacesRef}
             fetchDetails
+            onBlur={() => {
+                if (isSelected) {
```

Comment:
From the travis failure it looks like we prefer an early return here -- https://github.com/Shopify/web-configs/blob/main/packages/eslint-plugin/docs/rules/prefer-early-return.md